### PR TITLE
Update set-the-caller-id-for-a-user.md

### PR DESCRIPTION
**Pull request by shashank1320**
Use of Powershell for resource account related information in note section.
> You need to use the PowerShell cmdlets for Resource account number if not visible in drop down.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/set-the-caller-id-for-a-user.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ By default, the following caller ID settings are **turned off**. This means that
     
   > [!NOTE]
   > If you want to use the *Service* parameter, you must specify a valid service number.
+  > You need to use the PowerShell cmdlets for Resource account number if not visible in drop down.
```
Comment (JohanFreelancer9) [inline Teams/set-the-caller-id-for-a-user.md:36]:
```suggestion
  > - If you want to use the *Service* parameter, you must specify a valid service number.
  > - You need to use the PowerShell cmdlets New-CsCallingLineIdentity or Set-CsCallingLineIdentity in the Teams PowerShell module 2.3.1 or later for the Resource account number if it is not visible in the dropdown.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Teams/set-the-caller-id-for-a-user.md (hunk 1 above)
Comment (JohanFreelancer9) [reply Teams/set-the-caller-id-for-a-user.md:36]: @shashank1320 - I am really not sure about this - but your change seems incomplete. Please commit my suggested change if it is correct or edit your suggestion directly to provide complete information.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Teams/set-the-caller-id-for-a-user.md (hunk 1 above)
Comment (shashank1320) [reply Teams/set-the-caller-id-for-a-user.md:36]: Updated with suggestion.

**Pull request merged**